src/build_log.cc

```diff
@@ -353,7 +353,7 @@ BuildLog::LogEntry* BuildLog::LookupByOutput(const string& path) {
 }
 
 bool BuildLog::WriteEntry(FILE* f, const LogEntry& entry) {
-  return fprintf(f, "%d\t%d\t%d\t%s\t%" PRIx64 "\n",
+  return fprintf(f, "%d\t%d\t%" PRId64 "\t%s\t%" PRIx64 "\n",
```

Comment:
Unrelated to this change, but older ninjas will be pretty confused if they read build logs created by the new ninja. This isn't a new problem, but the `if (log_version < kOldestSupportedVersion` above should probably use `!=` instead. Not in this change, mostly talking to myself.)